packages/main/src/themes/StepInput.css

```diff
@@ -20,12 +20,65 @@
 	text-align: inherit;
 }
 
-:host(:not([value-state]):not([readonly]):not([disabled]):hover),
-:host([value-state=None]:not([readonly]):not([disabled]):hover) {
-    background-color: var(--sapField_Hover_Background);
-    border: 1px solid var(--sapField_Hover_BorderColor);
+:host(:not([value-state]):not([readonly]):not([disabled])){
+	box-shadow: none;
 }
 
+:host([value-state=Success]:not([readonly]):not([disabled])) {
+	background-color: var(--sapField_SuccessBackground);
+}
+
+:host([value-state=Error]:not([readonly]):not([disabled])) {
+	background-color: var(--sapField_InvalidBackground);
+}
+
+:host([value-state=Information]:not([readonly]):not([disabled])) {
+	background-color: var(--sapField_InformationBackground);
+}
+
+:host([value-state=Warning]:not([readonly]):not([disabled])) {
+	background-color: var(--sapField_WarningBackground);
+}
+
+:host(:not([value-state]):not([readonly]):not([disabled]):not([focused]):hover),
+:host([value-state=None]:not([readonly]):not([disabled]):not([focused]):hover) {
+	background-color: var(--sapField_Hover_Background);
+	border: var(--_ui5_step_input_border_style_hover);
+	box-shadow: var(--_ui5-input-hover-box-shadow);
+}
+
+:host([value-state=Success]:not([readonly]):not([disabled]):not([focused]):hover) {
+	box-shadow: var(--_ui5-input-value-state-success-hover-box-shadow);
+}
+
+:host([value-state=Information]:not([readonly]):not([disabled]):not([focused]):hover) {
+	box-shadow: var(--_ui5-input-value-state-information-hover-box-shadow);
+}
+
+:host([value-state=Warning]:not([readonly]):not([disabled]):not([focused]):hover) {
+	box-shadow: var(--_ui5-input-value-state-warning-hover-box-shadow);
+}
+
+:host([value-state=Error]:not([readonly]):not([disabled]):not([focused]):hover) {
+	box-shadow: var(--_ui5-input-value-state-error-hover-box-shadow);
+}
+
+:host([value-state=Success]:not([readonly]):not([disabled]):hover),
+:host([value-state=Error]:not([readonly]):not([disabled]):hover),
+:host([value-state=Information]:not([readonly]):not([disabled]):hover),
+:host([value-state=Warning]:not([readonly]):not([disabled]):hover) {
+	background-color: var(--_ui5_step_input_button_state_hover_backgroun_color);
```

Comment:
Are you sure that the parameter resolves the correct value, since there is a typo

Reply:
done